client/hotrod-client/src/main/java/org/infinispan/client/hotrod/configuration/ConfigurationChildBuilder.java

```diff
@@ -195,6 +195,16 @@ public interface ConfigurationChildBuilder {
     */
    ConfigurationBuilder maxRetries(int maxRetries);
 
+   /**
+    * The time for a failed server to be cleared allowing for it to attempt to reconnect at a later point.
+    * <p>
+    * If the value is less than or equal to 0 it will be disabled, meaning a failed SERVER will not be reconnected to
+    * until all configured servers have failed or a topology update (TOPOLOGY_AWARE and CONSISTENT_HASH_AWARE only)
```

Comment:
do you mean `HASH_DISTRIBUTION_AWARE`? :) 
```suggestion
    * until all configured servers have failed or a topology update (TOPOLOGY_AWARE and HASH_DISTRIBUTION_AWARE only)
```

Reply:
Yes, just misremembered the name. I also am updating them to be references so it can't happen again :)